eng/versioning/version_client.txt

```diff
@@ -146,7 +146,7 @@ com.azure:azure-messaging-eventhubs-checkpointstore-jedis;1.0.0-beta.2;1.0.0-bet
 com.azure:azure-messaging-eventhubs-stress;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-eventhubs-track1-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-eventhubs-track2-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-messaging-servicebus;7.14.4;7.14.5
+com.azure:azure-messaging-servicebus;7.14.5;7.15.0-beta.1
```

Comment:
seems the script did not check the latest beta version before generating a new beta version.

Reply:
Right, I've opened a PR to use the correct beta.4 version. I'll close this auto PR.